### PR TITLE
Fix NFT sizes and add docs

### LINKED ISSUE #94: Fix inconsistent NFT sizes
Looks like when we added gif and video support, it broke consistent sizing for the component:
image

I'll take on either resizing the content to fit in the box when it's a square, or adding padding around it so that we have consistency in sizing.

### PR DESCRIPTION
**Pull request by etr2460**
Closes #94 

## Description
Attempts to ensure all nft cards are squares of the same size. Also adds some more docs

## Screenshot
Screen Shot 2021-12-17 at 10 09 29 PM
Screen Shot 2021-12-17 at 10 09 34 PM
Screen Shot 2021-12-17 at 10 09 41 PM

to: @Dhaiwat10

### CODE REVIEW

File: packages/components/src/components/NFT/NFT.tsx [hunk 1]
```diff
@@ -54,8 +64,12 @@ export const NFT = ({ contractAddress, tokenId }: NFTProps) => {
           animationUrl: data.animation_url,
         });
       }
-    } catch (error: any) {
-      setErrorMessage(error.message);
+    } catch (error) {
+      if (error instanceof Error) {
```
Comment (maximebonhomme) [inline packages/components/src/components/NFT/NFT.tsx:68]: Could be good to create a helper for this as we're handling errors in quite a few places. What do you reckon? Can add a separate ticket for this.

Comment (etr2460) [reply packages/components/src/components/NFT/NFT.tsx:68]: Makes sense. It seems like we're handling errors (and the typing of them) in a couple ways. I'll make an issue to figure out the best way to handle errors in our code, and use whatever helper function i make across the codebase

Comment (etr2460) [reply packages/components/src/components/NFT/NFT.tsx:68]: https://github.com/Developer-DAO/web3-ui/issues/133

Comment (Dhaiwat10) [reply packages/components/src/components/NFT/NFT.tsx:68]: @maximebonhomme feel free to create that ticket. Good suggestion

**Comment by Dhaiwat10** [comment]
@etr2460 looks good. Can you add a changeset by running `yarn changeset`? I think we need a minor bump for the components package

**Comment by etr2460** [comment]
Thanks for adding the changeset @Dhaiwat10. do you have docs around when to do a minor vs. a patch changeset? This PR was mainly a fix, although it did add a new (optional) prop to the NFT component

**Comment by Dhaiwat10** [comment]
@etr2460 no I don't think we have any proper documentation around that. Do you think we need it?

**Pull request merged**